Utils/hdinsight-node-common/Test/java/com/microsoft/azure/hdinsight/spark/common/CosmosServerlessSparkBatchJobScenario.kt

```diff
@@ -71,6 +84,13 @@ class CosmosServerlessSparkBatchJobScenario {
         doReturn(submissionParameter).`when`(serverlessJobMock).getSubmissionParameter()
         doReturn(connectUri).`when`(serverlessJobMock).connectUri
         doReturn(Observable.just(true)).`when`(serverlessJobMock).prepareSparkEventsLogFolder()
+        doAnswer { invocation ->
+            val args = invocation.arguments
+            println(args[0] as String)
+        }.`when`(serverlessJobMock).ctrlInfo(ArgumentMatchers.anyString())
```

Comment:
Is there another way to avoid invoking `ctrlInfo()`?

Reply:
mock `getCtrlSubject` method